Gems/AzQtComponentsForPython/CMakeLists.txt

```diff
@@ -0,0 +1,224 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+cmake_minimum_required(VERSION 3.20)
+
+# The name of the Python import module we're generating.
+set(BINDINGS_LIBRARY "azqtpyside")
```

Comment:
This doesn't look like a Gem, it looks like a 3rdParty library.

- It doesn't look like the Default Gem template was used to create this structure.
- There are no CMake Targets with [GEM_MODUL](https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/Code/CMakeLists.txt#L124)E target property to hook into the Gem module loading system
- There is a hook into the Module loading system with an AZ::Module derived class that uses the [AZ_DECLARE_MODULE_CLASS](https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/Code/Source/Tools/%24%7BName%7DEditorModule.cpp#L47) macro
- This has a completely different structure for any other gem we have within the O3DE repo.